examples/apps/aml/standalone-cipher/signature.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2019 Banco Santander S.A.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import json
+import random
+import logging
+
+import encryptionAlg
+from Crypto.Cipher import PKCS1_OAEP
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA
+
+logger = logging.getLogger(__name__)
+#No of bytes of encrypted session key to encrypt data
+NO_OF_BYTES = 16
+
+#############################
+#############################
+class ClientSignature(object) :
+    
+    def __init__(self):
+        self.private_key = None
+        self.public_key = None
+        self.param_pool = ["requesterNonce", "workOrderId", "workerId", "requesterId","inData"]
+
+#---------------------------------------------------------------------------------------------
```

Comment:
Many functions in this file are same as that in https://github.com/hyperledger/avalon/blob/master/examples/common/python/utility/signature.py. I think it make sense to reuse it than duplicating it. How about deriving from ClientSignature class in examples/common/python/utility/signature.py